js/nodes/solver_settings/structural_mechanic_solver.js

```diff
@@ -0,0 +1,54 @@
+class StructuralMechanicSolver {
```

Comment:
```suggestion
class StructuralMechanicsSolver {
```
Same for the filename.

Reply:
Done